Sources/HttpParser.swift

```diff
@@ -40,7 +40,7 @@ public class HttpParser {
         guard url.endIndex > queryStart else {
             return []
         }
-        let query = String(url[queryStart..<url.endIndex])
+        let query = String(url[queryStart..<url.endIndex])!
```

Comment:
Force cast is bad way

Reply:
Here's a better solution https://github.com/httpswift/swifter/pull/321